webrtc.html

```diff
@@ -1293,6 +1293,27 @@
                     method was invoked.</p>
                   </li>
                   <li>
+                    <p>Let <var>offerBundle</var> be an implementation specific
+                    collection of information that captures the changes made to
+                    the media configuration of <var>connection</var> since the
+                    last time an offer or answer was created, or since the
+                    creation of <var>connection</var> if no offer or answer has
+                    been created prior to this call. The following pieces of
+                    information MUST be captured:</p>
+                    <ul>
+                      <li>
+                        <p>If <code><a href=
+                        "#dom-rtcpeerconnection-createdatachannel">createDataChannel()</a></code>
```

Comment:
Perhaps this should be in terms of SctpTransport, which is like the data channel version of RtpTransceiver.